lace/tests/cli.rs

```diff
@@ -533,6 +533,7 @@ mod run {
             .arg(dirname)
             .output()
             .expect("failed to execute process");
+        dbg!(&output);
```

Comment:
Not that it hurts anything--this is a test--but did you mean to leave this `dbg!`?